From 70dfe49feaa69d2bf5cab64b6f43a51b5b5da656 Mon Sep 17 00:00:00 2001
From: Hien Huynh <hien.huynh.px@renesas.com>
Date: Wed, 2 Dec 2020 11:27:57 +0700
Subject: [PATCH 191/358] arm64: dts: renesas: r8a774e1: Add the emergency node
 in thermal-zones

This patch enables emergency shutdown support for r8a774e1 SoC.

Signed-off-by: Hien Huynh <hien.huynh.px@renesas.com>
---
 arch/arm64/boot/dts/renesas/r8a774e1.dtsi | 14 ++++++++++++++
 1 file changed, 14 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a774e1.dtsi b/arch/arm64/boot/dts/renesas/r8a774e1.dtsi
index bbac62d..4f8c863 100644
--- a/arch/arm64/boot/dts/renesas/r8a774e1.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a774e1.dtsi
@@ -2651,6 +2651,20 @@
 	};
 
 	thermal-zones {
+		emergency {
+			polling-delay = <1000>;
+			on-temperature = <110000>;
+			off-temperature = <95000>;
+			target_cpus =	<&a57_1>,
+					<&a57_2>,
+					<&a57_3>,
+					<&a53_0>,
+					<&a53_1>,
+					<&a53_2>,
+					<&a53_3>;
+			status = "disabled";
+		};
+
 		sensor_thermal1: sensor-thermal1 {
 			polling-delay-passive = <250>;
 			polling-delay = <1000>;
-- 
2.7.4

